zmon_worker_monitor/builtins/plugins/time_.py

```diff
@@ -30,12 +33,20 @@ def create(self, factory_ctx):
 
 
 class TimeWrapper(object):
-    def __init__(self, spec='now', utc=False):
-        now = (datetime.utcnow() if utc else datetime.now())
+
+    EPOCH = datetime.fromtimestamp(0, pytz.UTC)
+
+    def __init__(self, spec='now', utc=False, tz_name=None):
+        if utc and tz_name:
+            raise ValueError('Ambiguous time zone. Do not use "utc" and "tz_name" parameter at the same time.')
+
+        tz = pytz.timezone(tz_name) if tz_name else None
+        self.timezone = pytz.UTC if utc else tz if tz_name else tzlocal.get_localzone()
```

Comment:
I think it would be more readable if statement/block syntax (instead of inline) was used.

Reply:
I can fix it